lfs/client.go

```diff
@@ -178,6 +182,9 @@ func DownloadCheck(oid string) (*objectResource, error) {
 		return nil, Error(err)
 	}
 
+	io.Copy(ioutil.Discard, res.Body)
+	res.Body.Close()
```

Comment:
We shouldn't need this. `res` comes from `doLegacyApiRequest()`, which [calls `decodeApiResponse()`](https://github.com/github/git-lfs/blob/b0318b25350b9abfe56af0afb01913a9b2ed56a4/lfs/client.go#L413). That function [parses json and immediately closes the response body](https://github.com/github/git-lfs/blob/b0318b25350b9abfe56af0afb01913a9b2ed56a4/lfs/client.go#L594-L595). While it doesn't hurt, these lines shouldn't be required.